src/components/molecules/card/card.html

```diff
@@ -0,0 +1,16 @@
+<div class="card">
+  <h5 class="card-title">CONCEPT</h5>
+  <a href="#" class="card-wrapper">
+    <img
+      src="../../../img/name=concept.png"
+      class="card-img-top"
+      alt="カフェの内観"
+    />
+  </a>
+  <div class="card-body">
+    <p class="card-text">説明文</p>
+    <a href="#" class="button">
+      <p class="concept">コンセプト</p>
+    </a>

```

Comment:
pタグを使わなくても、これで大丈夫です！
```suggestion
    <a href="#" class="button">
      コンセプト
    </a>
```